cockatrice/src/window_sets.cpp

```diff
@@ -125,14 +125,14 @@ WndSets::WndSets(QWidget *parent) : QMainWindow(parent)
     labNotes->setWordWrap(true);
     labNotes->setTextInteractionFlags(Qt::TextBrowserInteraction);
     labNotes->setOpenExternalLinks(true);
-    labNotes->setText(
-        "<b>" + tr("Deck Editor") + ":</b> " +
-        tr("Only cards in enabled sets will appear in the deck editor card list") + "<br><br>" + "<b>" +
-        tr("Card Art") + ":</b> " + tr("Image priority is decided in the following order") + "<ol><li>" + tr("The") +
-        "<a href='https://github.com/Cockatrice/Cockatrice/wiki/"
-        "Custom-Cards-%26-Sets#to-add-custom-art-for-cards-the-easiest-way-is-to-use-the-custom-folder'> " +
-        tr("CUSTOM Folder") + "</a></li><li>" + tr("Enabled Sets (Top to Bottom)") + "</li><li>" +
-        tr("Disabled Sets (Top to Bottom)") + "</li></ol>");
+    labNotes->setText("<b>" + tr("Deck Editor") + ":</b> " +
+                      tr("Only cards in enabled sets will appear in the card list of the deck editor") + "<br><br>" +
+                      "<b>" + tr("Card Art") + ":</b> " + tr("Image priority is decided in the following order") +
+                      "<ol><li>" + "CUSTOM " + tr("Folder") +
```

Comment:
This won't work for a lot of languages, eg. italian, where you put the name of things at the end of the sentence (Cartella CUSTOM)

Reply:
I expected that to be a problem with Arabic. Thanks for the information!

Looks like we have to keep the hard coded name in the translation or insert it via %1